applications/ecallmgr/src/ecallmgr_app.erl

```diff
@@ -72,3 +76,17 @@ declare_exchanges() ->
 node_bindings() ->
     _ = kz_nodes_bindings:bind('ecallmgr', ?MODULE),
     'ok'.
+
+-spec freeswitch_nodesup_bind() -> 'ok'.
+freeswitch_nodesup_bind() ->
+    _ = kazoo_bindings:bind(<<"freeswitch.node.modules">>, ?MODULE, 'freeswitch_node_modules'),
+    'ok'.
+
+-spec freeswitch_nodesup_unbind() -> 'ok'.
+freeswitch_nodesup_unbind() ->
+    _ = kazoo_bindings:unbind(<<"freeswitch.node.modules">>, ?MODULE, 'freeswitch_node_modules'),
+    'ok'.
+
+-spec freeswitch_node_modules() -> ne_binaries().
+freeswitch_node_modules() ->
+    application:get_env(?APP, 'node_modules', ?NODE_MODULES).
```

Comment:
Do we really need to fetch these when the application is not started?
Cause these node modules are now specified in 2 different places without even a comment saying "hey don't forget to keep this in sync with that other list"